### ISSUE TITLE
Navigating to new page doesn't reset scroll position

### ISSUE THREAD

**Issue by patrickledew** [Vito-Research/Vito-Website#27]
**To recreate:**
- Scroll down on home page
- Click Contributors or Blog in the navbar
- The new page will be scrolled as well

**Comment by patrickledew** [comment]
@deznorth Issue still persists when clicking footer links, might be better to call restoreScroll whenever a page component is mounted?

**Comment by patrickledew** [comment]
Even better if we could do it for every page, [](https://stackoverflow.com/questions/36904185/react-router-scroll-to-top-on-every-transition) might be helpful

**Comment by deznorth** [comment]
good catch, I completely neglected the footer. Making a new PR 👍

**Comment by patrickledew** [comment]
Merged PR, still had some issues with it not scrolling all the way up when navigating to the contributors page. I think this is something to do with the `behavior: "smooth"` on the scrollTo call, maybe the contributors page is half loaded when it's called and then it stops when the rest of it loads? For now I just set the behavior to auto and it works. ([Commit](https://github.com/Vito-Research/Vito-Website/commit/1dacf3f4bac35527f9286a879e9468ec27894c2f))
